.travis.yml

```diff
@@ -0,0 +1,44 @@
+dist: trusty
+env:
+  global:
+    - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
+jobs:
+  include:
+    - stage: common
+      name: "Verify common files are consistent"
+      language: javascript
+      # Only check on pull requests to master
+      if: branch = master AND type = pull_request
+      script:
+        - git remote add -f b https://github.com/w3c/json-ld-wg.git
+        - git remote update
+        - git diff --exit-code remotes/b/master -- common
+    - stage: examples
+      name: "Verify examples are consistent"
+      language: ruby
+      rvm: 2.6
+      script: "bundle exec rake test"
+    - stage: links
+      name: "Check internal links and HTML"
+      sudo: required
+      services:
+        - docker
+      # Because (alas) rawgit is end of life, we'll run this only on the master branch.
+      if: branch = master
+      language: ruby
+      rvm: 2.6
+      script:
+        - docker pull stupchiy/checklink
+        - docker run -it stupchiy/checklink -b -X '(http|https|mailto):' "https://labs.w3.org/spec-generator/?type=respec&url=https://rawgit.com/w3c/json-ld-streaming/${TRAVIS_BRANCH}/index.html?specStatus=WD&shortName=json-ld11-streaming"
+        - bundle exec rake check_html
+    - stage: echidna
+      name: "Echidna Publication"
+      language: javascript
+      # Only on push to "publications" branch
+      if: branch = publications AND type = push
+      env:
+        - URL="https://w3c.github.io/json-ld-streaming/ECHIDNA"
+        - DECISION="https://www.w3.org/2018/json-ld-wg/Meetings/Minutes/2018/2018-09-28-json-ld#resolution2"
+        - secure: "lLdok36PZzMuoXtVGDLtqEPXscvbM/lAg3LUZ9rrFysqflIGZYbA4A19u7J9abB9Z5OLyOlUGbIoTxENbeO8A0lEsEjwYqh+Hg0FL2ZLMR8xHmjfQ+wiO/q8RURogaun1FZXIm1chaYarf2V2QBmyxpzX3+Hqhaw4oji16NrPh+tcdZVRoER24kV6AeDowuS9ZXviZP3fqQhRYG114lUY6H94GdqYWGvVrRjA6z8bpyHB+yDD+8n7ma0MNiufbhBzeq1uz57TJebsr8gSs8rp7VH8w3NI3Gf6Ig5XdhOGcFssZNm5YWY/KaS5aihYoAvMJguKQ5ALARvJzpzmdMBz9KsBRKBgtPW56r2XD1ITlFpo3qVRxLsyGTu8y3hWY6U8SqMdpdYF1dwHLfoL6vO5ATn/CZ3B4POmSW5FepwOcOAluLE6LwBgd35h8Fypt4g5ruceZyFddIcvAYlUBPZPIi/tItQVz6zz+xb8aL7vR2/ekLKFTckbskb/gN60fsIjUumxQeOslw9mGO6jUch67eizzaL2Wob6VKDrxoymb873LbepTgOFFksaUp8lwbtsUPgOtJQ7hWCH2StZmIioh4LMumcRKm0EcIdtn5vh/f1MDETVmLpgKt0HeMkaH6bUCe6km3ZedaRfObLqEV3+tVyo4T4f+u0eo2S/JBFGOU="
+      script:
+        - curl "https://labs.w3.org/echidna/api/request" --data "url=$URL" --data "decision=$DECISION" --data "token=$TOKEN" --data "editorial=true"
```

Comment:
As far as I remember, the "editorial=true" was relevant for CR only. Not for a WG Note

Reply:
```suggestion
        - curl "https://labs.w3.org/echidna/api/request" --data "url=$URL" --data "decision=$DECISION" --data "token=$TOKEN"
```